bin/release

```diff
@@ -109,16 +114,16 @@ bin/aws-eb-docker/release-eb-version.sh "v$VERSION"
 # echo "ok"
 
 cd ..
-
-# heroku it
+    
+echo "pulling down metabase-deploy"
 if ! [ -d "metabase-deploy" ]; then
+    echo "cant find ... cloning"
     git clone git@github.com:metabase/metabase-deploy.git
 fi
 
+echo "pulling"
 cd metabase-deploy
-git fetch
-git co "origin/master"
+git pull
 
+echo "release heroku artifacts"
 bin/release-heroku "v$VERSION"
-
-# technologic
```

Comment:
side-chaining low-pass filters to the kicks is still the business. how would you do trance music with no side-chaining ⛓